IBF-Typhoon-model/src/climada/hazard/trop_cyclone.py

```diff
@@ -88,7 +88,8 @@ class TropCyclone(Hazard):
         'SP' Southern Pacific
         'SA' South Atlantic
     """
-    intensity_thres = 17.5
+    # TODO: make a PR in Climada to make this an option
+    intensity_thres = 10   # Default 17.5
```

Comment:
in the same PR please change the filter at 
https://github.com/rodekruis/Typhoon-Impact-based-forecasting-model/blob/master/IBF-Typhoon-model/src/climada/hazard/tc_tracks_forecast.py#L147